README.md

```diff
@@ -854,6 +855,35 @@ datetime
 
 ```
 
+
+### Social Dominance
+
+Returns the % of the social dominance a given project has over time in a given social channel. Available sources are: PROFESSIONAL_TRADERS_CHAT, TELEGRAM, DISCORD, REDDIT, ALL.
+
+[Premium metric](#premium-metrics)
+
+```python
+san.get(
+    "social_dominance/bitcoin",
```

Comment:
All examples should use `santiment` as a slug. The reason is that we have time restrictions for all slugs except santiment.

Our data is freely available for `santiment` but for the other projects some of the data (this query for example) is limited to 3 months historical data. Leaving `bitcoin` will make the example not working in a few weeks.